src/views/Config/FileTransfer.vue

```diff
@@ -286,6 +286,13 @@ const handleSchemaToFileInfo = (data: SchemaData, type: 'local' | 's3') => {
 
     const fieldKeyFileSchema = get(components, getFieldSchemaPath(getSSLConfPath('keyfile')))
     fieldKeyFileSchema.componentProps = { placeholder: t('Base.keyFilePlaceholder') }
+
+    const fieldCiphersSchema = get(components, getFieldSchemaPath(getSSLConfPath('ciphers')))
+    fieldCiphersSchema.items.component = 'input'
```

Comment:
input-textarea?

Reply:
https://github.com/emqx/emqx-dashboard5/blob/327024cb059b6485530408654ce26347f7931ca1/src/components/SchemaForm.tsx#L365

emmmmm, this was called `input` before, maybe it is better to get input-textarea🤔️